extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -495,7 +498,10 @@ export class AmpA4A extends AMP.BaseElement {
         .then(returnedArray => returnedArray[0]);
       });
     }))
-    .then(returnedArray => returnedArray[0]);
+    .then(returnedArray => {
```

Comment:
Seems this is the same concern as I mentioned earlier:  cost of including then just for emitLifecycleEvent.  Can this not be included in a previous portion of the promise chain?